packages/desktop-electron/index.ts

```diff
@@ -183,29 +188,28 @@ async function createWindow() {
   clientWin = win;
 }
 
-function isExternalUrl(url) {
+function isExternalUrl(url: string) {
   return !url.includes('localhost:') && !url.includes('app://');
 }
 
-function updateMenu(budgetId) {
+function updateMenu(budgetId?: string) {
   const isBudgetOpen = !!budgetId;
-  const menu = getMenu(isDev, createWindow);
+  const menu = getMenu(isDev, createWindow, budgetId);
   const file = menu.items.filter(item => item.label === 'File')[0];
-  const fileItems = file.submenu.items;
+  const fileItems = file.submenu?.items || [];
   fileItems
     .filter(item => item.label === 'Load Backup...')
     .forEach(item => {
       item.enabled = isBudgetOpen;
-      item.budgetId = budgetId;
```

Comment:
Typescript complained at this - it was a bit hacky. I'm passing budgetid to the getMenu method instead